test/aws/scaleup.yml

```diff
@@ -120,20 +117,3 @@
       --timeout=0s
     with_items: "{{ pre_scaleup_workers_name }}"
 
-  - name: remove existing machinesets
-    k8s:
-      api_version: machine.openshift.io/v1beta1
-      kubeconfig: "{{ kubeconfig_path }}"
-      namespace: openshift-machine-api
-      kind: MachineSet
-      name: "{{ item }}"
-      state: absent
-    with_items: "{{ pre_scaleup_machineset_names }}"
-
-  - name: Delete CoreOS nodes
-    k8s:
-      kubeconfig: "{{ kubeconfig_path }}"
-      kind: Node
-      name: "{{ item }}"
-      state: absent
-    with_items: "{{ pre_scaleup_workers_name }}"
```

Comment:
Did you intend to drop these lines, remove and drop CoreOS nodes?

Reply:
No. Thank you. Fixing now.